test/unit/test_ipmi_console.py

```diff
@@ -130,10 +124,6 @@ def setUpClass(cls):
         node = CNode(node_info)
         node.init()
         node.precheck()
-        node.start()
```

Comment:
suggest leave a comment here like below, instead of removing them.
```
# comment next line since this test case doesn't need to start node actually.
# node.start()
```